packages/fxa-auth-server/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     // Permit, but do not report errors for JS files
     // TODO: Remove after transition to TS is complete
     "checkJs": false,
-    "types": ["accept-language", "mocha", "mozlog", "node"]
+    "types": ["accept-language", "mocha", "mozlog", "node"],
+    "esModuleInterop": true
```

Comment:
`esModuleInterop` is already true in `tsconfig.base.json`. why here?

Reply:
I was getting some typescript errors when running tests through webstorm...which now I can't seem to reproduce. From what I recall it was trying to use fs to load config and failed because it couldn't find the import. I'll revert this it seems to be working now 🤷🏽‍♂️